docs/api/en/textures/CompressedArrayTexture.html

```diff
@@ -49,10 +48,15 @@ <h2>Properties</h2>
 
 		<h3>[property:number wrapR]</h3>
 		<p>
-		This defines how the texture is wrapped in the depth direction.<br />
-		The default is [page:Textures THREE.ClampToEdgeWrapping], where the edge is clamped to the outer edge texels.
-		The other two choices are [page:Textures THREE.RepeatWrapping] and [page:Textures THREE.MirroredRepeatWrapping].
-		See the [page:Textures texture constants] page for details.
+			This defines how the texture is wrapped in the depth direction.<br />
+			The default is [page:Textures THREE.ClampToEdgeWrapping], where the edge is clamped to the outer edge texels.
+			The other two choices are [page:Textures THREE.RepeatWrapping] and [page:Textures THREE.MirroredRepeatWrapping].
+			See the [page:Textures texture constants] page for details.
+		</p>
+
+		<h3>[property:Image image]</h3>
```

Comment:
> [property:Image image]

That should be `[property:Object image]`.

Reply:
That is clearer.  I was following what is currently in DataTexture and DataArrayTexture. Update all 4 ?

Reply:
Yes please. The property is definitely not of type `Image`.